build.sbt

```diff
@@ -1,17 +1,24 @@
-scalaVersion in ThisBuild := "2.12.3"
+inThisBuild(List(
+  scalaVersion := "2.12.3",
+  organization := "org.scalameta",
+  version := "0.1-SNAPSHOT"
+))
 
-lazy val LanguageServer = RootProject(
-  uri(
-    "git://github.com/gabro/dragos-vscode-scala.git#formatting"
+lazy val languageserver = project
```

Comment:
may want to exclude this project from .scalafmt.conf

```
project.git = true
project.excludeFilters = [
   languageserver/src/main/scala
]

Reply:
I've already done it in the Travis PR